test/sanity/validate-modules/schema.py

```diff
@@ -17,29 +17,33 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from voluptuous import ALLOW_EXTRA, Any, Required, Schema
+from ansible.compat.six import string_types
+# Because string_types from six is tuple mostly used in isinstance()
+# function and we need just class name for schema definition
+string_types = string_types[0]
```

Comment:
I feel like my preference here would be to not overwrite the same variable, it can cause confusion.

Instead maybe don't do this at all, and instead use `*string_types` everywhere instead.